src/content/tutorials/observability/multi-tenancy/_index.md

```diff
@@ -1,15 +1,201 @@
 ---
 title: Multi-tenancy
-description: Guides explaining how to configure multi-tenancy in the Observability Platform.
+description: Understanding multi-tenancy concepts and best practices for the Observability Platform.
```

Comment:
```suggestion
description: How multi-tenancy works in the Observability Platform and best practices.
```

This doc also shows a lot how to do and configure things which IMO should be kept for the `Step-by-step guide`